src/elements/hx-select-control/index.spec.js

```diff
@@ -0,0 +1,66 @@
+import { fixture, expect } from '@open-wc/testing';
+
+/**
+ * <hx-select-control> component tests
+ *
+ * @type HXSelectControlElement
+ *
+ */
+describe('<hx-select-control> component tests', () => {
+    const template = '<hx-select-control>';
+    const mockup = `
+          <hx-select-control>
+              <select
+                id="selDemo"
+              >
+              <!-- ... -->
+              </select>
+            <hx-select></hx-select>
+            <label
+              for="selDemo"
+            >
+            Choose an Option
+            </label>
+          </hx-select-control>`;
```

Comment:
Please align the mockup properly (lint indentations).

Reply:
Changes have been done, Please review